packages/nameguard-sdk/src/index.ts

```diff
@@ -689,6 +689,8 @@ export class NameGuard {
    *
    * Returns `display_name` to be shown to users and estimates `impersonation_status`
    *
+   * If `address` has a primary name and `options.returnNameGuardReport` is `True`, then NameGuard will attempt to inspect that primary name. Else, NameGuard will not perform any primary name inspection and the returned `nameguard_result` field will be `null`.
```

Comment:
```suggestion
   * If `address` has a primary name and `options.returnNameGuardReport` is `true`, then NameGuard will attempt to inspect that primary name. Else, NameGuard will not perform any primary name inspection and the returned `nameguard_result` field will be `null`.
```

In JS / TS we write boolean constants in all lowercase.